hera_mc/cm_hookup.py

```diff
@@ -368,8 +416,31 @@ def is_fully_connected(self, hookup_dict, any_or_all='all'):
         else:
             return num_fully_connected > 0
 
+    def write_hookup_to_file(self, hookup_dict):
+        np.save(self.hookup_local_file, hookup_dict)
+        np.save(self.part_type_cache_file, self.part_type_cache)
```

Comment:
The two arrays can be written to a single file thusly:

```python
with open(self.hookup_local_file, 'wb') as f:
    np.save(f, hookup_dict)
    np.save(f, self.part_type_cache)
```

And analogous code (changing the "wb" to "rb" to open the file in read mode) will read them back out.